docs/using-radicle/running-a-seed-node.md

```diff
@@ -3,5 +3,12 @@ id: running-a-seed-node
 title: Running a seed node
 ---
 
-This is a link to [another document.](doc3.md)  
-This is a link to an [external page.](http://www.example.com)
+To improve data availability, participants in the network can choose to act as
+seeds. This is similar in concept to a pub in Secure Scuttlebutt: seeds are
+always-on nodes that automatically track discovered projects, thereby
+increasing the availability of these projects on the network.
+
+To set up your own seed node follow the [instructions][seed] in the
```

Comment:
Is there a reason we link through instead of having it rewritten here? I think it would be nice to have the commands laid on it the GitHub repo shown in the docs, instead of linking someone away.

@rudolfs @xla ?

Reply:
The only reason I did it this way, was to keep the docs up-to-date. If we have the same info in 2 places it'll go out of sync very fast.

I could write a more detailed version on how to set it up here, complete with example outputs of the commands. So users can more easily debug if something goes wrong.

Reply:
Added a more extended explanation.